crates/autopilot/src/run.rs

```diff
@@ -607,43 +606,34 @@ pub async fn run(args: Arguments) {
             .instrument(tracing::info_span!("order_events_cleaner")),
     );
 
-    if args.enable_colocation {
-        if args.drivers.is_empty() {
-            panic!("colocation is enabled but no drivers are configured");
-        }
```

Comment:
We should still keep this panic.

Reply:
Reverted.
Initially I removed it because the check should be done on config loading but anyway, let's keep it until we are there.